sycl/test/fpga_tests/fpga_aocx_win.cpp

```diff
@@ -0,0 +1,60 @@
+//==--- fpga_aocx_win.cpp - AOT compilation for fpga using aoc with aocx ---==//
```

Comment:
The source code in this test completely duplicates fpga_aocx.cpp. Can we get rid of that duplication?

Reply:
Different, Windows-specific `RUN` lines are required here. I'm not sure if there's a way to execute *some* of the LIT commands on a particular OS, while the rest remain OS-agnostic.

Reply:
We could have two LIT tests reference the same source file, as it is done in device code split LIT tests.